store.go

```diff
@@ -776,6 +779,10 @@ func (s *store) GraphOptions() []string {
 	return s.graphOptions
 }
 
+func (s *store) PullOptions() map[string]string {
+	return s.pullOptions
```

Comment:
This should be returning a copy so that callers can't mess with the configuratoin by modifying the map's contents.